### ISSUE TITLE
session_value is not an alias for account_session_value

### ISSUE THREAD

**Issue by ayghor** [jeremyevans/rodauth#27]
i just noticed this inspecting teh sauce. teh docs says, on

https://github.com/jeremyevans/rodauth/blame/master/doc/base.rdoc:199
```
session_value :: Alias for account_session_value.
```

but

https://github.com/jeremyevans/rodauth/blob/master/lib/rodauth/features/base.rb:143
```
    def account_id
      account[account_id_column]
    end
    alias account_session_value account_id

    def session_value
      session[session_key]
    end
    alias logged_in? session_value
```

so if you have account set, but not session, teh values would diverge?

**Comment by jeremyevans** [comment]
Good catch, I'll update the documentation.

**Comment by jeremyevans** [comment]
Fixed by 18fd0011fe6817d67f02e1534bf799a34ff8f68e
